tests/integration/utils/testprogram.py

```diff
@@ -764,7 +765,19 @@ def shutdown(self, signum=signal.SIGTERM, timeout=10):
                         break
                     raise
                 if datetime.now() > future:
-                    raise TimeoutError('Timeout waiting for "{0}" pid'.format(pid))
```

Comment:
This change should have gone into one of the other patches or into its own patch.  Oh well.